packages/storage-evm/scripts/base-mainnet-contracts-scripts/analyze-cash-flows.ts

```
@@ -0,0 +1,543 @@
+        2,
+      )} USDC cents`,
+    );
+    console.log(
+      `  Discrepancy: ${((totalHouseholdBalance + importBalance) / 100).toFixed(
+        2,
+      )} USDC cents`,
+    );
+
+    const discrepancy = totalHouseholdBalance + importBalance;
+
+    if (Math.abs(discrepancy) > 100) {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'discrepancy'. Consider to delete it or rename it.

'discrepancy' is defined here:


(lint/suspicious/noRedeclare)